Docs/topics/discord/faq.md

```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
 
 Not at the moment. PPSSPP support is in the works, but is currently not possible.
 
-## Q: I followed the PSP guide, but my game wont connect!
+## Q: I followed the PSP guide, but my game wont connect, how do I fix it?
```

Comment:
Old way reads better tbh

Thinking of dropping the `Q: ` portion

Reply:
For now i'll just revert that one line, revamps to the whole file should probably be done in a different patchest